### PR TITLE
tests: migration: make flaky test more verbose

### PR DESCRIPTION
**Pull request by jean-edouard**
**What this PR does / why we need it**:
This test flaked, but all the info we got from the failure was that false != true...
Adding more verbiage.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/migration_test.go [hunk 1]
```diff
@@ -3090,7 +3090,7 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				By("Verifying at least once that both pods are protected")
 				for _, pod := range pods.Items {
 					err := virtClient.CoreV1().Pods(vmi.Namespace).EvictV1beta1(context.Background(), &policyv1beta1.Eviction{ObjectMeta: metav1.ObjectMeta{Name: pod.Name}})
-					Expect(errors.IsTooManyRequests(err)).To(BeTrue())
+					Expect(errors.IsTooManyRequests(err)).To(BeTrue(), "expected TooManyRequests error, got: %v", err)
```
[the remark this answers is not in the document]
Comment (fossedihelm) [reply tests/migration_test.go:3093]: This can be helpful since there are other occurrences in `migration_test` file

**Comment by fossedihelm** [comment]
/lgtm
/hold
@jean-edouard Place an hold to give time to handle suggestions in the review. Feel free to unhold

File: tests/migration_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (jean-edouard) [reply tests/migration_test.go:3093]: I actually don't want to eat the error, I want to make sure there was an error and it was that one...

[the remark this answers is not in the document]
Comment (xpivarc) [reply tests/migration_test.go:3093]: Right so for this case, I don't have any other suggestions.

**Pull request merged**